### PR TITLE
Updates windows installer default lib location

### PR DESCRIPTION
**Pull request by LogicalPhallacy**
You can use the emby import to move an existing library this way.

### CODE REVIEW

**Comment by joshuaboniface** [comment]
Can we just have it call the migrate script if it detects a move? I.e. in BASH as an example:

```
if [[ ! -d ${new_dir} && -d ${old_dir} ]]; then
    mv ${old_dir} ${new_dir}
    run_db_schema_migrate.sh
fi
```

My Powershell skill is basically nonexistent :D

File: deployment/windows/install-jellyfin.ps1 [hunk 1]
```diff
@@ -82,7 +82,7 @@ if($Quiet.IsPresent -or $Quiet -eq $true){
     }else{
         $Script:InstallServiceAsUser = $true
         $Script:UserCredentials = $ServiceUser
-        $Script:JellyfinDataDir = "C:\Users\$($Script:UserCredentials.UserName)\Appdata\Roaming\jellyfin\"}
+        $Script:JellyfinDataDir = "$env:HOMEDRIVE\Users\$($Script:UserCredentials.UserName)\Appdata\Local\jellyfin\"}
```
Comment (EraYaN) [inline deployment/windows/install-jellyfin.ps1]: Can we make this LOCALAPPDATA too? Or is it not defined for service users?

Comment (LogicalPhallacy) [reply deployment/windows/install-jellyfin.ps1]: LOCALAPPDATA is defined by the context of the running process. If you're in that section of the code you're using service credentials which aren't going to necessarily be those of the user running the install script.

**Comment by LogicalPhallacy** [comment]
> Can we just have it call the migrate script if it detects a move? I.e. in BASH as an example:
> if [[ ! -d ${new_dir} && -d ${old_dir} ]]; then
>     mv ${old_dir} ${new_dir}
>     run_db_schema_migrate.sh
> fi
> 
> My Powershell skill is basically nonexistent :D

We actually should be alright upon examining the database structure. Environment variables are what is stored in the paths in the database. I don't know when that change was made though, if legacy paths are hardcoded in the old schema we may have to indicate that a metadata rescan will be required after a migration or modify that code. Dynamically detecting the old path and replacing it is going to be beyond the scope of the installer I'm afraid.

**Comment by joshuaboniface** [comment]
@LogicalPhallacy Can you rebase this off the release branch?

**Pull request merged**